plugin/storage/cassandra/schema.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+CQLSH_HOST=${CQLSH_HOST:-"cassandra"}
+CASSANDRA_WAIT_TIMEOUT=${CASSANDRA_WAIT_TIMEOUT:-"60"}
+DATACENTER=${DATACENTER:-"openshift"}
+KEYSPACE=${KEYSPACE:-"jaeger_v1_${DATACENTER}"}
+
+total_wait=0
+while true
+do
+  ping -c 1 ${CQLSH_HOST} > /dev/null 2>&1
+  if (( $? == 0 )); then
+    break
+  else
+    if (( total_wait >= ${CASSANDRA_WAIT_TIMEOUT} )); then
+      echo "Timed out waiting for Cassandra."
+      exit 1
+    fi
+    echo "Cassandra is still not up at ${CQLSH_HOST}. Waiting 1 second."
+    sleep 1s
+    ((total_wait++))
+  fi
+done
+
+echo "Generating the schema for the keyspace ${KEYSPACE} and datacenter ${DATACENTER}"
+export KEYSPACE
+
+# the `test` parameter is to force the script to use a SimpleStrategy instead of
```

Comment:
Ok, that's it then. I'm not that familiar with Cassandra and noticed that I could change the script to `test` and it would work :-) My deployment script creates 2 Cassandra nodes, but it seems it was still not enough to avoid this error.

Should we move all these Cassandra-related tasks to a new task?